src/js/gi/components/Checkbox.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import _ from 'lodash';
+
+import ToolTip from '../../common/components/form-elements/ToolTip';
+
+
+/**
+ * A form checkbox with a label that can display error messages.
```

Comment:
For what it's worth, the `value` and `dirty` pattern is there to support showing validation errors only after a field has been interacted with (i.e. it's dirty). Definitely not needed if you don't have required or error messages.

Those components were originally built for HCA and forms like what the Rainbows team is building, but we're moving away from using these particular components now (the schema form work). So there's probably some room in the future to have a simpler set of components that fit more cleanly in apps that just have a few input fields.